scripts/docs/build.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
+export DD_TRACE_ENABLED=0
+
 set -eux
 
 reno lint
-sphinx-build -W -b spelling docs docs/_build/html
-sphinx-build -W -b html docs docs/_build/html
+sphinx-build -vvv -W -b spelling docs docs/_build/html
```

Comment:
do we want to keep this verbosity? (I don't care either way, just checking that it was intentional)

Reply:
yeah that's intentional. it makes debugging easier.